jurisdiction/README.md

```diff
@@ -0,0 +1,181 @@
+| ---------------- | --------- | --- | ----------------------------------------------------------------------------------- |
+| `jurisdiction_id`| UUID      | Required   | Unique ID of Jurisdiction. This field is immutable. |
+| `agency_key`     | String    | Required   | A unique string key for the Jurisdiction. This field must also be immutable. Allows for easier management of Jurisdiction-based access control in JWTs. |
+| `agency_name`    | String    | Optional   | Human-readable agency name for display purposes. |
+| `description`    | String    | Required   | Description of Jurisdiction.                                                               | 
+| `geography_id`   | UUID      | Optional   | The unique ID of the geography covered by this Jurisdiction.  |
+| `mobility_modes`  | String[]    | Optional   | Set this field to specify what mobility modes a jurisdiction applies to. Valid values are left up to the agency to determine, but a couple examples that we think might be useful are 'taxi' and 'micromobility'. An empty array or undefined field indicates all mobility modes are covered under this jurisdiction. |
+| `timestamp`      | timestamp | Required   | Creation or update time of a Jurisdiction.                                                 |
+
+Formatted in JSON, a Jurisdiction object should look like this:
+
+```
+{
+  jurisdiction_id: UUID
+  agency_key: string
+  agency_name: string
+  geography_id: UUID
+  timestamp: Timestamp
+}
+```
+
+## Endpoints
+
+### GET /Jurisdictions
+
+Gets all of an agency's Jurisdictions.
+
+Parameters:
+| Name         | Type      | R/O | Description                                    |
+| ------------ | --------- | --- | ---------------------------------------------- |
+| `effective`   | Timestamp | O   | See the state of all the Jurisdictions (i.e. which ones are effective) at that point in time. If not supplied, the default is to show only Jurisdictions that are currently in effect.     |
+
+Response codes:
+- 200 - success
+- 403 - unauthorized
+- 500 - server error
+
+### GET /Jurisdictions/:jurisdiction_id
+
+Gets a single Jurisdictions.
+
+Parameters:
+| Name         | Type      | R/O | Description                                    |
+| ------------ | --------- | --- | ---------------------------------------------- |
+| `effective`   | Timestamp | O   | See the version of the Jurisdiction that was in effect at that point in time.      |
+
+Response codes:
+- 200 - Success
+- 403 - Unauthorized
+- 404 - not found
+- 500 - Server error
+
+[Flat files](#flat-files)
+To use flat files, Jurisdictions shall be represented in the following file:
+
+- `jurisdictions.json`
+- `geographies.json`
+
+The publishing Agency should establish and communicate to interested parties how frequently these files should be polled.
+
+The `updated` field in the payload wrapper should be set to the time of publishing a revision, so that it is simple to identify a changed file.
+
+#### Example `jurisdictions.json`
```

Comment:
For consistency across MDS this should really be in its own sub folder, and linked to in a section on this page, like with [geography](https://github.com/openmobilityfoundation/mobility-data-specification/blob/dev/geography/examples/README.md)